docs/docs/en/guide/installation/kubernetes.md

```diff
@@ -12,7 +12,7 @@ If you are a new hand and want to experience DolphinScheduler functions, we reco
 
 ## Install DolphinScheduler
 
-Please download the source code package `apache-dolphinscheduler-<version>-src.tar.gz`, download address: [download address](/en-us/download/download.html)
+Please download the source code package `apache-dolphinscheduler-<version>-src.tar.gz`, download address: [download address](https://dolphinscheduler.apache.org/#/en-us/download)
```

Comment:
TBH, use the whole url will broken the doc check when we remove to other place, but maybe is the best way currently

Reply:
It seems that the link is unavailable.